src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -137,8 +137,12 @@ public ActionId getActionId(BlockActionPayload blockActionPayload) {
 
     private TextObject getTextObject(String title, String content) {
         StringBuilder sb = new StringBuilder();
-        sb.append(title + ":\n");
-        sb.append("*" + content + "*");
+        if (title != null) {
```

Comment:
Maybe do a StringUtils.isNotEmpty check instead? If the content is empty string, you also don't want to include.